README.md

```diff
@@ -0,0 +1,15 @@
+# org.srb2.SRB2Kart-galaxy_azerty
+
+Flatpak for SRB2Kart-Galaxy with the AZERTY french keyboard layout.
```

Comment:
can't that be built into the original flatpak?

Reply:
It's an hard edit of the source code as most of the SRB2Kart exe mods are. It replaces the hard coded keybindings within the depths of the doom engine to emulate AZERTY kb.

So I'm not sure I understand this question…